micro_manager/micro_manager.py

```diff
@@ -427,7 +426,7 @@ def initialize(self) -> None:
         self._micro_sims_init = False  # DECLARATION
 
         # Read initial data from preCICE, if it is available
-        initial_data = self._read_data_from_precice()
+        initial_data = self._read_data_from_precice(dt=self._micro_dt)
```

Comment:
@BenjaminRodenberg what is the correct `dt` value to use when reading initial data from preCICE?

Reply:
`dt=0`, if you want to read the data at the beginning of the window.